docs/tutorial-osparc.ipynb

```diff
@@ -0,0 +1,341 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# o²S²PARC module from the NIH SPARC Python Client\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this document you will learn how to access the [o²S²PARC Python Client](https://itisfoundation.github.io/osparc-simcore-clients/#/) as part fo the [NIH SPARC Python Client](https://github.com/nih-sparc/sparc.client).\n",
+    "\n",
+    "### Pre-requites:\n",
+    "- Python version >=3.10: as requested by the [sparc.client](https://pypi.org/project/sparc.client/) package.\n",
+    "- An account on [osparc.io](https://osparc.io): you will need this to generate API tokens to use the o²S²PARC module. If you don't have an account, request one at o²S²PARC Support, as explained [here](https://sparc.science/resources/4LkLiH5s4FV0LVJd3htsvH).\n",
+    "- Have a look to the [NIH SPARC Python Client doc](https://github.com/nih-sparc/sparc.client/blob/main/docs/tutorial.ipynb) as a preliminary step (and learn how to set up the Configuration)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Installation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import the necessary packages\n",
+    "import importlib, getpass\n",
+    "try:\n",
+    "  import sparc.client\n",
+    "except ImportError:\n",
+    "  ! pip install sparc.client\n",
+    "try:\n",
+    "  from tqdm import tqdm\n",
+    "except ImportError:\n",
+    "  ! pip install tqdm\n",
+    "from pprint import pprint\n",
+    "import os\n",
+    "from tqdm import tqdm\n",
+    "from sparc.client import SparcClient\n",
+    "from sparc.client.services.o2sparc import (\n",
+    "  O2SparcService,\n",
+    "  O2SparcSolver\n",
+    ")\n",
+    "from time import sleep\n",
+    "from pathlib import Path\n",
+    "from tempfile import TemporaryDirectory"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Login to the o²S²PARC Python Client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Set-up credentials for o²S²PARC in osparc.io as explained [here](https://docs.osparc.io/#/docs/platform_introduction/user_setup/security_details?id=generating-o%c2%b2s%c2%b2parc-tokens). Then use the \"Key\" as \"Username\" and \"Secret\" as \"Password\". Another option is to export them as environment variables."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.environ[\"O2SPARC_HOST\"] = getpass.getpass(\"osparc host, e.g. https://api.osparc.io\")\n",
```

Comment:
Add `:` to end of `osparc host, e.g. https://api.osparc.io` to match other getpass commands.

Reply:
Good point. Thanks a lot, I have performed this change.